springwolf-core/src/main/java/io/github/stavshamir/springwolf/schemas/DefaultSchemasService.java

```diff
@@ -25,10 +27,17 @@ public class DefaultSchemasService implements SchemasService {
     private final Map<String, Schema> definitions = new HashMap<>();
     private Map<String, Schema> finalizedDefinitions = null;
 
-    public DefaultSchemasService(List<ModelConverter> externalModelConverters, ExampleGenerator exampleGenerator) {
+    public DefaultSchemasService(
+            List<ModelConverter> externalModelConverters,
+            ExampleGenerator exampleGenerator,
+            SpringwolfConfigProperties properties) {
 
         externalModelConverters.forEach(converter::addConverter);
         this.exampleGenerator = exampleGenerator;
+
```

Comment:
After the swagger core update, getfqn can be called to fetch the current flag status which allows the reset to the original value